rasa/utils/tensorflow/data_generator.py

```diff
@@ -3,15 +3,15 @@
 import logging
 import scipy.sparse
 import numpy as np
-import tensorflow as tf
 
 from rasa.utils.tensorflow.constants import SEQUENCE, BALANCED
 from rasa.utils.tensorflow.model_data import RasaModelData, Data, FeatureArray
+from keras.utils import data_utils
 
 logger = logging.getLogger(__name__)
 
 
-class RasaDataGenerator(tf.keras.utils.Sequence):
+class RasaDataGenerator(data_utils.Sequence):
```

Comment:
Had to do this because otherwise DataHandler's `select_data_adapter(x, y)` step (from `CustomDataHandler`) fails to select the correct adapter (`KerasSequenceAdapter`) `isinstance(x, data_utils.Sequence)` is false

Reply:
I am able to train a model when `RasaDataGenerator` inherits from `tf.keras.utils.Sequence`. I tried on moodbot. Can you mention how to reproduce this?

Reply:
That's weird.. I definitely did this because it was crashing on me, but now I can't reproduce. Changed it back.